packages/react-data-grid/src/RowsContainer.tsx

```diff
@@ -1,8 +1,10 @@
+import React from 'react';
+
 export interface RowsContainerProps {
   id: string;
   children: React.ReactNode;
 }
 
 export default function RowsContainer({ children }: RowsContainerProps) {
-  return children;
+  return <>{children}</>;
```

Comment:
TS complains that components must return a `ReactElement`, and a `ReactNode` is not good enough.